### DISCUSSION TITLE
Callout improvements for revealjs

### DISCUSSION THREAD

**Discussion by garthtarr** [quarto-dev/quarto-cli#727] [Q&A]
I love the callouts but I've encountered a couple of issues using them with revealjs presentations:

- For the `"simple"` appearance there's an icon rendering issue (wrong size)
- The collapsible option doesn't work
- This is a little niche but it would be nice to be able to have a title only callout box, particularly for presentations. There's some distracting white space even when the content is empty. A version of the collapsible option (without the drop down arrow) would be great.

<img width="681" alt="Screen Shot 2022-04-25 at 9 56 42 pm" src="https://user-images.githubusercontent.com/2189134/165084427-5416b643-4840-4705-9b80-24fbded7da0c.png">

```
---
title: "Presentation"
format: revealjs
---

## Slide with callout

::: {.callout-note appearance="simple"}

## Simple appearance incorrect icon sizing.

Icon size incorrect.

:::

:::{.callout-caution collapse="true"}
## With option `collapse="true"` (but not collapsible)

This is an example of a 'folded' caution callout that can be expanded by the user. 
You can use `collapse="true"` to collapse it by default or `collapse="false"` to 
make a collapsible callout that is expanded by default.
:::

:::{.callout-note}

## Title only (note the unecessary white space)

:::
```

**Comment by jjallaire** [comment]
Hi there, thanks for the feedback! We were able to fix the 1st and 3rd issues here: https://github.com/quarto-dev/quarto-cli/commit/8f14b229334d750872749591351c58b01f75bf6f

In terms of collapse/expand, that is currently a Bootstrap only feature.

**Reply by garthtarr** [reply]
Outstanding, thanks so much for the super quick fix - tested on the dev version and working perfectly.

**Comment by mcanouil** [comment]
FYI, "collapsible callout in Reveal.js" is tracked in #1328.
